model_compression_toolkit/keras/quantizer/gradient_ptq/weight_quantizer.py

```diff
@@ -22,6 +22,14 @@
 from model_compression_toolkit.keras.quantizer.base_quantizer import BaseTrainableQuantizer
 from model_compression_toolkit.keras.quantizer.gradient_ptq.utils import symmetric_constrained_quantizer
 from model_compression_toolkit.common.constants import THRESHOLD
+from model_compression_toolkit.common.defaultdict import DefaultDict
+
+
+def get_kernel(weights_list):
+    for w in weights_list:
+        if "kernel" in w.name:
+            return w
+    raise Exception("Can't find kernel variable")
```

Comment:
Typehints, comments, and please use the constant "kernel" from keras constants

Reply:
Or maybe the kernel attribute should be taken from fw_info... What do you think?